java/integTest/src/test/java/glide/SharedClientTests.java

```diff
@@ -52,7 +52,7 @@ public static void teardown() {
     }
 
     @SneakyThrows
-    @ParameterizedTest
+    @ParameterizedTest(autoCloseArguments = false)
```

Comment:
Why? don't we want to close the client after each test?

Reply:
No, we use it in multiple tests and close manually. There are no plans to change the test approach.